tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -177,6 +195,42 @@ Embed data block into release content
     user clicks button    Add data block
     user chooses and embeds data block    ${DATABLOCK_NAME}
 
+Check footnote is displayed in content Tab
+    # workaround for lazy loaded content blocks (scroll up 210px)
+    user scrolls down    -210
```

Comment:
This is a bit naff given that we're currently looking at the section that we've just dropped a data block on :laughing:

We might want to see if there's another way to automatically render the data block as soon as it's embedded.

Reply:
Done. I've replaced this with: 
```robot
user checks accordion section contains x blocks    ${CONTENT_SECTION_NAME}    1    id:releaseMainContent

user scrolls to element    id:releaseMainContent
```